pkg/commands/git_commands/commit.go

```diff
@@ -22,6 +22,11 @@ func (self *CommitCommands) RewordLastCommit(message string) error {
 	return self.cmd.New("git commit --allow-empty --amend --only -m " + self.cmd.Quote(message)).Run()
 }
 
+// Reset the author of the topmost commit.
```

Comment:
The convention is to start the comment with the name of the element it describes, in this case that would be `ResetAuthor reset the author of the topmost commit`